css/styles.css

```diff
@@ -534,19 +619,23 @@ img {
   align-items: center;
 }
 
-/*.phase-logo {
-    width: 24px;
-    height: 24px;
-}*/
+.map-describe .read-more-btn {
+    margin: 0;
+}
 
 .map-pagination {
   display: flex;
-  gap: 124px;
+  
+  gap: 19px;
+  justify-content: space-between;
 }
 
 .map-pagination-btn {
   background-color: transparent;
   border: none;
+
+  position: relative;
+  top: -15px;
```

Comment:
Don't use` position: relative` for moving elements. If you want to move this button to top of parent element add `align-items: flex-start` to `.map-pagination`